src/storage/index/IndexPolicyMaker.cpp

```diff
@@ -151,6 +153,81 @@ bool IndexPolicyMaker::exprEval(Getters &getters) {
     }
     return true;
 }
+
+RelationalExpression::Operator
+IndexPolicyMaker::reversalRelationalExprOP(RelationalExpression::Operator op) {
+    switch (op) {
+        case RelationalExpression::Operator::LT: {
+            return RelationalExpression::Operator::GT;
+        }
+        case RelationalExpression::Operator::LE: {
+            return RelationalExpression::Operator::GE;
+        }
+        case RelationalExpression::Operator::GT: {
+            return RelationalExpression::Operator::LT;
+        }
+        case RelationalExpression::Operator::GE: {
+            return RelationalExpression::Operator::LE;
+        }
+        default : {
+            return op;
+        }
+    }
+}
+
+bool IndexPolicyMaker::writeScanItem(const std::string& prop, const OperatorItem& item) {
```

Comment:
Please add some comments about this method.  It looks some bit complex.

Reply:
Added. please take a look in source code. thanks.